packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -164,3 +167,40 @@ class _AlternateCheckedModeBanner extends StatelessWidget {
     );
   }
 }
+
+class _BulletSpacer extends StatelessWidget {
```

Comment:
common_widgets.dart could be a good place for this class along with a TODO to use the correct color.

Reply:
done